### PR TITLE
Omer Afzal's Portfolio

### PR DESCRIPTION
**Pull request by omer-creator**
This is my personal Portfolio.

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @omer-creator 👋 , 

Good job with this milestone so far!
There are just few more issues to work on before moving onto the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,267 @@
...
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md:10]: Image logo has been added

File: README.md [hunk 2]
```diff
@@ -1,2 +1,267 @@
...
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio Microverse] <a name="about-project"></a>
+
+> Repository for building a personal portfolio based on figma ui design.
+
+**portfolio_microverse** is a portfolio
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md]: Commented lines removed and edited README template in a more decriptive statements.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,267 @@
...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> We use html,css(flexbox,grid) to convert ui from figma to html,css.
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md]: Commented lines removed

File: README.md [hunk 4]
```diff
@@ -1,2 +1,267 @@
...
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md]: Readme template edited according to the project.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,267 @@
...
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md]: Commented lines removed

File: README.md [hunk 6]
```diff
@@ -1,2 +1,267 @@
...
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply README.md]: Unused content removed.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+                    <li>
+                        <a class="nav-text-links" href="#about">About</a>
+                    </li>
+                    <li>
+                        <a class="nav-text-links" href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </nav>
+
+        </header>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply index.html:50]: Overlapping content fixed.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+</head>
+
+<body>
+    <main class="main-bg-headline">
+        <header class="display-on-mobile">
+            <!-- Starting Tool bar-->
+            <nav class="nav">
+                <div>
+                    <p class="Subhead welcome-text">Welcome to my Portfolio</p>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply index.html:23]: Typo Fixed.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+                    <a href="https://twitter.com/Micronaut_Omer"><img src="img/twitter.svg" alt="twitter" /></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/muhammad-omer-afzal-67ab17250/"><img src="img/linkedin.svg"
+                            alt="Linkedin" /></a>
+                </li>
+                <li>
+                    <a href="https://github.com/omer-creator"><img src="img/github.svg" alt="github" /></a>
+                </li>
+            </ul>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply index.html:87]: Added two more social media content links

File: index.html [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+        </section>
```
[the remark this answers is not in the document]
Comment (omer-creator) [reply index.html:88]: All suggested and required change completed.

**Review by didierganthier — COMMENTED** [review]
Hi @omer-creator ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project with the StyLelint (linters) test not passing 😢 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Review by christianbiring1 — APPROVED** [review]
Hi @omer-creator , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
congraaaa

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Good looking UI :art: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**